### PR TITLE
Add SeatGeek to Events section

### PR DESCRIPTION
**Pull request by jeffrand**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by fintzd — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -411,6 +411,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Eventbrite](https://www.eventbrite.com/developer/v3/) | Find events | `OAuth` | Yes | Unknown |
 | [Picatic](http://developer.picatic.com/?utm_medium=web&utm_source=github&utm_campaign=public-apis%20repo&utm_content=toddmotto) | Sell tickets anywhere | `apiKey` | Yes | Unknown |
+| [SeatGeek](https://platform.seatgeek.com/) | Search events, venues and performers | `apiKey` |  Yes  | Unknown |
```
Comment (matheusfelipeog) [inline README.md:414]: Each segment must start and end with exactly 1 space

```suggestion
| [SeatGeek](https://platform.seatgeek.com/) | Search events, venues and performers | `apiKey` | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contriibution @jeffrand 😄 

Can you resolve this review, please?

**Review by matheusfelipeog — APPROVED** [review]
Wow, you were fast. We appreciate it, thank you very much.

**Comment by jeffrand** [comment]
No problem!

**Pull request merged**